src/types/cursor/xcursor_manager.rs

```diff
@@ -0,0 +1,38 @@
+use wlroots_sys::{wlr_xcursor_manager, wlr_xcursor_manager_create, wlr_xcursor_manager_load, wlr_xcursor_manager_set_cursor_image};
+use types::Cursor;
+use std::ptr;
+use utils::safe_as_cstring;
+
+#[derive(Debug)]
+pub struct XCursorManager {
+    manager: *mut wlr_xcursor_manager
+}
+
+impl XCursorManager {
+    pub fn create<T: Into<Option<String>>>(name: T, size: u32) -> Option<Self> {
+        unsafe {
+            let name_str = name.into().map(safe_as_cstring);
+            let name_ptr = name_str.map(|s| s.as_ptr()).unwrap_or(ptr::null_mut());
+            let manager = wlr_xcursor_manager_create(name_ptr, size);
```

Comment:
Need corresponding call to `wlr_xcursor_manager_destroy` in `drop` method implemented by manually implementing the `Drop` trait on `XCursorManager`.